canonical-web-and-design/snapcraft.io
# Move listing form fields into partials

## Done
Moved listing form fields into partials to make easier to work with in preparation for metadata link fields

## QA
- Go to https://snapcraft-io-4026.demos.haus/notion-snap/listing
- Check that the form fields still work

## Issue
https://github.com/canonical-web-and-design/marketplace-tribe/issues/2606
